### PR TITLE
chore: Document the cli

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: src/tooling/grain_cli.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```sh
+grain run hello.gr.wasm
+```
+
+It should be noted that `grain run` is used here on the WebAssembly file produced via `grain compile`.
+
+## `grain doc`
+
+Produces documentation for a Grain program. See the full [Graindoc page](./graindoc) to learn more!
```
Comment (phated) [inline src/tooling/grain_cli.md:68]: Can you add the usage examples like done in `grain format`? Sometimes you just want a reference guide for running on CLI

Comment (phated) [reply src/tooling/grain_cli.md:68]:
````suggestion
Produces documentation for a Grain program. See the full [Graindoc page](./graindoc) to learn more!

### Example

Generating docs for a single file to stdout:

```sh
grain doc hello.gr
```

Generating docs for a single file:

```sh
grain doc hello.gr -o hello.md
```

Generating docs for an entire project:

```sh
grain format . -o .
```
````

Something like this?

Comment (ospencer) [reply src/tooling/grain_cli.md:68]: LGTM!

File: src/tooling/grain_cli.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+Generating docs for a single file:
+
+```sh
+grain doc hello.gr -o hello.md
+```
+
+Generating docs for an entire project:
+
+```sh
+grain format . -o .
```
Comment (phated) [inline src/tooling/grain_cli.md:87]:
```suggestion
grain doc . -o .
```
I'm a dummy

**Review by phated — APPROVED** [review]
Looks good!

**Pull request merged**